src/main/kotlin/com/terraformation/backend/seedbank/api/SummaryController.kt

```diff
@@ -68,17 +65,32 @@ class SummaryController(
   fun summarizeAccessionSearch(
       @RequestBody payload: SummarizeAccessionSearchRequestPayload
   ): SummarizeAccessionSearchResponsePayload {
+    val originalCriteria = payload.toSearchNode(accessionsPrefix)
+
+    // If there are fuzzy search criteria, first see if we get results by treating them as exact
```

Comment:
With these finer details.. should we create a summary service for testability?

Reply:
Yeah, I was being lazy here, but you're right. Will refactor this. We definitely want to stick with our "controllers are dumb wrappers around services and have so little logic that we're okay with minimal test coverage" approach which this definitely violates.